controller/AdminAccessController.php

```diff
@@ -145,11 +150,38 @@ public function savePermissions()
         }
 
         return $this->returnJson([
-            'success'   => $success,
-            'message'   => $message
+            'success' => $success,
+            'message' => $message
         ], $code);
     }
 
+    /**
+     * Find users to assign access rights
+     */
+    public function findUser()
+    {
+            $params = $this->getGetParameter('params');
+            $query = $params['query'];
+            $userService = $this->getServiceLocator()->get(UserService::SERVICE_ID);
+            $data = [];
+            foreach ($userService->findUser($query) as $user) {
+                $labels = $user->getPropertyValues(OntologyRdfs::RDFS_LABEL);
+                $data[] = [
+                    'id'                     => $user->getIdentifier(),
+                    OntologyRdfs::RDFS_LABEL => empty($labels) ? 'unknown user' : reset($labels)
```

Comment:
`reset` is common way to get label. I can't be sure that $labels will be array with numerical keys.
`unknown user` this is depends on realization. In our case it will be presented but i can't grant that all other realizations will support this.